addons/mail/static/src/new/core/user_settings_service.js

```diff
@@ -66,6 +64,15 @@ export class UserSettings {
         return constraints;
     }
 
+    getVolume(rtcSession) {
+        return (
+            rtcSession.volume ||
+            this.partnerVolumes.get(rtcSession.partnerId) ||
+            this.guestVolumes.get(rtcSession.guestId) ||
```

Comment:
Would be nice to reconcile `partnerVolumes` and `guestVolumes` to `volumes` (key being a persona)

Reply:
@alexkuhn  Sounds nice, I will keep that in mind when we start polishing the code